tests/core/lowering/test_remove_dropout_pass.cpp

```diff
@@ -32,39 +32,15 @@ TEST(LoweringPasses, RemoveDropoutLowersCorrectly) {
   ASSERT_TRUE(!torch::jit::findPatternMatches(*tg, *sg).empty());
 }
 
-TEST(LoweringPasses, RemoveDropoutInplaceLowersCorrectly) {
-  std::string source_graph = R"IR(
-    graph(%x.1):
-      %3 : float = prim::Constant[value=0.5]()
-      %4 : bool = prim::Constant[value=0]()
-      %y.1 : Tensor = aten::dropout_(%x.1, %3, %4)
-      %11 : Tensor = aten::relu(%y.1)
-      return (%11))IR";
-  std::string target_graph = R"IR(
-    graph(%x.1):
-      %11 : Tensor = aten::relu(%x.1)
-      return (%11))IR";
-
-  torch_tensorrt::core::util::logging::get_logger().set_reportable_log_level(
-      torch_tensorrt::core::util::logging::LogLevel::kGRAPH);
-  auto sg = std::make_shared<torch::jit::Graph>();
-  torch::jit::parseIR(source_graph, sg.get());
-  torch_tensorrt::core::lowering::passes::RemoveDropout(sg);
-
-  auto tg = std::make_shared<torch::jit::Graph>();
-  torch::jit::parseIR(target_graph, tg.get());
-
-  ASSERT_TRUE(!torch::jit::findPatternMatches(*tg, *sg).empty());
-}
-
-TEST(LoweringPasses, RemoveFeatureDropoutLowersCorrectly) {
+TEST(LoweringPasses, RemoveDropoutNestedLowersCorrectly) {
   std::string source_graph = R"IR(
     graph(%x.1):
       %3 : float = prim::Constant[value=0.5]()
       %4 : bool = prim::Constant[value=0]()
-      %y.1 : Tensor = aten::feature_dropout(%x.1, %3, %4)
-      %11 : Tensor = aten::relu(%y.1)
-      return (%11))IR";
+      %y.1 : Tensor = aten::dropout(%x.1, %3, %4)
+      %z.1 : Tensor = aten::dropout(%y.1, %3, %4)
+      %12 : Tensor = aten::relu(%z.1)
+      return (%12))IR";
```

Comment:
This test case elicits the bug - using the original `RemoveDropout`, the lowering pass encounters a JIT assertion error due to dangling outputs.

Reply:
Do some of the clean up passes help with this? Like RemoveDeadCode

Reply:
No, other passes don't seem to help in this case because the error originates from within the `RemoveDropout` lowering pass itself. It seems that the graph is put into an inconsistent state during lowering, so when the rewrite pattern is run on the graph, it crashes entirely. Running `EliminateDeadCode` before the `RemoveDropout` lowering pass will not help, as the `aten::dropout` code is considered to be in-use, and running it after also does not work since the crash occurs within `RemoveDropout`.